Userland/Libraries/LibJS/Runtime/Temporal/AbstractOperations.cpp

```diff
@@ -660,7 +660,7 @@ ThrowCompletionOr<Value> to_relative_temporal_object(VM& vm, Object const& optio
         match_behavior = MatchBehavior::MatchMinutes;
 
         // See NOTE above about why this is done.
-        result = move(parsed_result.date_time);
+        result = move(parsed_result);
```

Comment:
The comment on line 615 is no longer true after this change, and then this one can be removed as well. I think we might just be able to name it `result` in the first place and get rid of this altogether?

Reply:
Yep! seems like we can get rid of that `parsed_result` business altogether. I'll make that change asap